### ISSUE TITLE
Missing hero element

### ISSUE THREAD

**Issue by AlzbetaT** [AusDTO/gov-au-beta#338]
@catherineedwards  

**Please add the hero element to these 3 pages:**

/categories/infrastructure-and-telecommunications
Should be: .hero-med

/departments
Should be: .hero-med

/ministers
Should be: .hero-med

**Comment by ltankey** [comment]
Thanks @AlzbetaT - what would be the h1 and p text that goes inside the hero-med?

**Comment by ltankey** [comment]
Should it also have the corporate tag? Or should it be grey? Could you post a picture of what you expect it to look like?

**Comment by catherineedwards** [comment]
Tracking as `SITES-529`

**Comment by TrebBrennan** [comment]
/categories/infrastructure-and-telecommunications
Remove existing H1 and P from content and move into `.hero-med`
```
<h1>Infrastructure and telecommunications</h1>
<p>Australian Government information and services related to infrastructure and 
telecommunications.</p>
```

/departments
Remove existing H1 from content and move into `.hero-med` with new copy
```
<h1>Departments</h1>
<p>The Australian Government has 18 departments and 189 agencies.</p>
```

/ministers
Remove existing H1 from content and move into `.hero-med` with new copy
```
<h1>Cabinet Ministers</h1>
<p>The Australian Government's ministers include Cabinet ministers, outer ministers, assistant ministers, ministers assisting and parliamentary secretaries.</p>
```

**Comment by ltankey** [comment]
Thanks @TrebBrennan  - should they have the corporate style or the grey?

**Comment by AlzbetaT** [comment]
@ltankey @TrebBrennan corporate hero

**Comment by tessereth** [comment]
Fixed in 1.0.0
